asciidoc/volume1/use-cases/tf1-ch-c-use-case-stad.adoc

```diff
@@ -35,16 +35,27 @@ image::../images/vol1-diagram-use-case-stad-tech-view.svg[align=center]
 *Then* The device will synchronize its time with the <<acronym_ts_service>>
 
 
-===== Scenario: <<acronym_stad>> {var_use_case_id}.2 - Device is connected to the MD LAN network with a TS Service and a user wants to change the device's time
+===== Scenario: <<acronym_stad>> {var_use_case_id}.1 - Device is connected to the MD LAN network and a user wants to change the device's time
 
-*Given* Device has detected at least one <<acronym_ts_service>>
-
-*When* The <<acronym_ts_service>> is operational
+*Given* Device is operational in <<acronym_md_lan>> network
 
-*And* The user attempts to change the time on the device manually
+*When*  The user attempts to change the time on the device manually
 
 *Then* The device will disable the ability to change its time manually
 
+==== Safety, Effectiveness & Security Considerations and Requirements
+
+.R1510
+[sdpi_requirement#r1510,sdpi_req_level=shall]
+****
+A <<vol1_spec_sdpi_p_actor_somds_participant>> shall not allow manual configuration of its internal clock.
```

Comment:
Should we really disallow manual clock configuration at all or just require participants to not connect to the MD LAN when the interal clock is configures manually?